en/docs/updates/authentication.md

```
@@ -0,0 +1,43 @@
+To explicitly clear a persisted session, unset the session environment variables before running the tool again, for example:
+
+```bash tab='Linux'
+unset SESSION_ACCESS_TOKEN SESSION_REFRESH_TOKEN SESSION_USERNAME
+```
+
+```bash tab='MacOS'
+unset SESSION_ACCESS_TOKEN SESSION_REFRESH_TOKEN SESSION_USERNAME
+```
+
+```powershell tab='Windows'
+$ Remove-Item Env:SESSION_ACCESS_TOKEN
```

markdownlint-cli2 (0.20.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)